lib/tarantool_server.py

```diff
@@ -464,14 +464,19 @@ def seek_wait(self, msg, proc=None, name=None, deadline=None):
                 break
             gevent.sleep(0.001)
         else:
+            color_stdout("\nLogfile {} does not exist\n".format(self.path),
+                         schema='error')
```

Comment:
1. Should we really write it 1000 times per second? How about a simple ratelimiting (add functions like `color_{stdout,log}_ratelimited()`?
2. I don't see a reason to show it on the terminal: it is legal if the log file will not appear during some time. How about logging with `color_log()`?

Reply:
3. Maybe it also worth to show how much time was spent and how much remains till the deadline.

Reply:
It will print the message only once when the loop will end.

Reply:
Point 2 also has no much sense, since we print the message only in case of the error.

Reply:
Point 3 is done.